examples/knative/powercli/kn-pcli-tag/handler.ps1

```diff
@@ -17,10 +17,16 @@ Function Process-Init {
    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor [System.Net.SecurityProtocolType]::Tls12 -bor [System.Net.SecurityProtocolType]::Tls13
 
    Write-Host "$(Get-Date) - Configuring PowerCLI Configuration Settings`n"
-   Set-PowerCLIConfiguration -InvalidCertificateAction:${VCENTER_CERTIFICATE_ACTION} -ParticipateInCeip:$true -Confirm:$false | Out-Null
+   Set-PowerCLIConfiguration -InvalidCertificateAction:${VCENTER_CERTIFICATE_ACTION} -ParticipateInCeip:$true -Confirm:$false
 
    Write-Host "$(Get-Date) - Connecting to vCenter Server $VCENTER_SERVER`n"
-   Connect-VIServer -Server $VCENTER_SERVER -User $VCENTER_USERNAME -Password $VCENTER_PASSWORD | Out-Null
+
+   try {
+      Connect-VIServer -Server $VCENTER_SERVER -User $VCENTER_USERNAME -Password $VCENTER_PASSWORD -ErrorAction Stop | Out-Null
```

Comment:
@embano1 Should we just remove the Out-Null all together?

Reply:
Hi William, I kept the Out-Null in because Dimitar said nothing against removing it. Maybe the reason was simply not thinking about the debugging case.

Reply:
There's no need for it, so lets just drop it to keep it clean

Reply:
Done